src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -23,27 +23,29 @@
 
     public static final Person ALICE = new PersonBuilder().withName("Alice Pauline")
             .withEmail("alice@example.com")
-            .withPhone("94351253")
+            .withPhone("94351253").withUserOwe("$0.00,2020-01-01")
```

Comment:
I would suggest shifting withUserOwe(..) down just to make the formatting nicer.